### ISSUE TITLE
Callout: High contrast issues

### ISSUE THREAD

**Issue by mikewheaton** [OfficeDev/office-ui-fabric-core#630]
We have two reported high contrast issues for the Callout:

1. There are no outlines around the callout:
 - ![image](https://cloud.githubusercontent.com/assets/1382445/15807656/5a205712-2b18-11e6-8be7-50d539700b9e.png)

2. Callout links are purple in HC Black. They should be yellow.
 - ![image](https://cloud.githubusercontent.com/assets/1382445/15807660/72423b76-2b18-11e6-9fba-3572d15bd70e.png)

**Comment by mikewheaton** [comment]
This issue was moved to OfficeDev/office-ui-fabric-js#55
